kie-soup-dataset/kie-soup-dataset-elasticsearch/pom.xml

```diff
@@ -35,11 +35,47 @@
           </excludes>
         </configuration>
       </plugin>
-      
+
     </plugins>
-    
+
   </build>
 
+
+  <dependencyManagement>
```

Comment:
Can you please move these to the dependencyManagement section in the kie-soup-parent POM with all the other third-party dependencies? Also please use properties for all the versions, following the convention of other property names (i.e. `version.org.apache.log4j`).